update_ip/services/cloudflare.py

```diff
@@ -9,13 +9,16 @@
 class CloudflareService(BaseDNSService):
     name = 'Cloudflare'
 
-    def __init__(self, email, api_key, domain, dns_type, **kwargs):
+    def __init__(self, email, api_key, dns_type, **kwargs):
```

Comment:
domain is fetched from update() function, and no, you can have multiple domains and subdomains with the same API key